bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -163,6 +165,7 @@ public class Clip2ThingHandler extends BaseThingHandler {
     private boolean updateLightPropertiesDone;
     private boolean updatePropertiesDone;
     private boolean updateDependenciesDone;
+    private boolean applyOffTransitionWorkaround;
```

Comment:
I suggest to rename this as `applyLkWiserOffTransitionWorkaround` so that it is clear to which products this patch would apply. (If there would be other models that would eventually need the same fix then we could rename the flag..)

Reply:
I actually named it like that on purpose to have an abstraction so that there would be no difference for Wiser vs. Osram for example:
- `updateProperties` checks for specific devices/models and sets up the flag.
- `handleCommand`/`applyDeviceSpecificWorkArounds `applies the work-around based on the flag, but without knowing the reason (because it doesn't need to).

I think this is cleaner. I made a mistake with the Wiser-specific comment though - I have moved it to `updateProperties` to complete the abstraction.

Reply:
Ok.